_includes/domain-search.html

```diff
@@ -0,0 +1,100 @@
+                                Read about next steps to take before requesting your .gov domain.
+                            </a>
+                        </p>
+                    </div>
+                </div>
+                `
+                )
+            } 
+            else {
+                render_result(`
+                <div class="usa-alert usa-alert--error usa-alert--slim" role="alert">
+                    <div class="usa-alert__body">
+                        <p class="usa-alert__text">
+                            <b>${requested_domain}.gov</b> isn't available.
+                            <a class="usa-link" href="{{ '/domains/choosing/' | url }}">
+                                Read more about choosing your .gov domain.
+                            </a>
+                        </p>
+                    </div>
+                </div>
+                `
+                )
+            }
+        }).catch(function(err) {
+            console.log('parsing failed', err);
+        })
+    }
+
+    function render_result(content, append = true){
+      document.getElementById('domain-available-message').innerHTML = (append == true) ? content : `<div></div>`;
+    }
+    
+</script>
+
+<section class="usa-section domain-section padding-bottom-5 bg-accent-cool-lighter">
+    <div class="grid-container">
+        <div class="domain-search-image">
+            <h2>
+                Start your .gov domain search
+            </h2>
+            <h3 class="domain-search-subtitle
+            line-height-heading-3
+            margin-top-0 margin-bottom-3
+            text-normal"
+            >
+                See if the .gov domain you want is available. We'll work with you
+                to find the best domain for your organization that meets our requirements.
+                <a href="{{ '/domains/choosing/' | url }}" class="usa-link">
+                    Read more about choosing your .gov domain.
+                </a>
+            </h3>
+            <form class="usa-search domain-search" role="search">
+                <label class="usa-sr-only" for="search-field-en-big">Search</label>
+                <input 
+                    class="usa-input usagov-search-autocomplete" 
+                    id="domain-input"
+                    type="search" 
+                    name="query" 
+                    autocomplete="off"
+                >
+                    <span class="usa-search__submit-text">.gov</span>
+                    <button 
+                    class="usa-button availability-button" 
+                    type="button"
+                    onclick="checkDomainAvailability()"
+                    onsubmit="return false">
+                        Check availability
+                    </button>
+            </form>
+            <div class="domain-alert" id="domain-available-message"></div>
+        </div>
+    </div>
+</section>
```

Comment:
add end of file newline please

Reply:
Whoa TIL I was like "why does a HTML file need a newline at the end" and it's bc if the last line in a file doesn't end with a newline then the addition of the next line affects 2 lines instead of 1 🤯